### DISCUSSION TITLE
Weird permission error

### DISCUSSION THREAD

**Discussion by Nisthar** [directus/directus#12540] [Archive]
I have 2 custom relationship fields on my `directus_users` table. 

![image](https://user-images.githubusercontent.com/11994157/161386713-4b1126b7-0d3c-40a7-be4b-e34f4807e8d1.png)

i want to hide the `sent_requests` and `received_requests` to be accessed by others. Only the current user should have access to this fields. (This would be a great addition to the permissions. that you can select which fields a certain role can access using dynamic variables)

The issue is that if i disable read access to `sent_requests` and `received_requests`. I can't do any PATCH updates on the user.
If i enable both the fields, i can do updates.

**Comment by Nisthar** [comment]
TLDR: Basically i need to give read access to both these fields to send update requests. Otherwise, i am getting `FORBIDDEN` error 

Both the fields are o2m relationship

**Comment by Nisthar** [comment]
Ok its not just the 2 fields. all the relational fields needs to have read access for updates to work.

**Comment by licitdev** [comment]
@Nisthar Thank you for your report. An issue has been filed. Ref: #12548
